### PR TITLE
"No messages" as empty state message added

### PR DESCRIPTION
**Pull request by jniederm**
screenshot:
image

---
This change is [Reviewable](https://reviewable.io/reviews/ovirt/ovirt-web-ui/435)

### CODE REVIEW

File: src/intl/messages.js [hunk 1]
```diff
@@ -117,6 +117,10 @@ export const messages = {
     description: 'About application',
   },
   logOut: 'Log out',
+  noMessages: {
+    message: 'No\u00A0messages',
```
Comment (bond95) [inline src/intl/messages.js]: I think here need to add some comment about \u00A0, because it is not very clear what this sign for.

Comment (jniederm) [reply src/intl/messages.js]: Do you think following line (122) is not enough?

Comment (bond95) [reply src/intl/messages.js]: Aha, yes, sorry, I didn't saw that.

**Comment by jniederm** [comment]
fixes #386

**Pull request merged**